### PR TITLE
Refactor CSS and Input Button

### PR DESCRIPTION
**Pull request by varun13169**

### CODE REVIEW

**Comment by itsmebipul00** [comment]
Good to merge!

**Review by zoef-yt — APPROVED** [review]
Good to merge.

File: library/input/input.css [hunk 1]
```diff
@@ -1,55 +1,60 @@
...
     width: 28.6rem;
     height: fit-content;
     position: relative;
 }
 
-.input-field__icon {
+.dui-inp-txt__icon {
+    bottom: 0;
     padding: 10px;
     min-width: 50px;
```
Comment (sudipkundu999) [inline library/input/input.css]: any specific reason for using px here?

**Review by sudipkundu999 — APPROVED** [review]
Good to merge

File: library/input/input.css [hunk 2]
```diff
@@ -1,55 +1,60 @@
...
     text-align: center;
 }
 
-.input-field__input {
+.dui-inp-txt__input {
     width: 100%;
     text-align: left;
```
Comment (developer-anand) [inline library/input/input.css]: remove spaces

**Review by Muzammil-khan710 — APPROVED** [review]
All good ! .. Good to merge

File: library/input/input.css [hunk 3]
```diff
@@ -58,16 +63,16 @@
 .input-field__icon--search {
```
Comment (developer-anand) [inline library/input/input.css]: for less than 1 rem you should use px

**Review by developer-anand — APPROVED** [review]
Good to Merge.

File: library/input/input.css (hunk 1 above)
Comment (varun13169) [reply library/input/input.css]: Corrected, thanks.

File: library/input/input.css (hunk 2 above)
Comment (varun13169) [reply library/input/input.css]: done.

File: library/input/input.css (hunk 3 above)
Comment (varun13169) [reply library/input/input.css]: No, that statement is just a recommendation and is true for properties you don't want to be responsive, ex: border-radius.

Here it is being used for positioning and changes with other components as well.

**Pull request merged**